Makefile

```diff
@@ -32,18 +30,10 @@ endif
 ifndef IMAGE_NAME
 $(error IMAGE_NAME is not set; check project.mk file)
 endif
-ifndef VERSION_MAJOR
-$(error VERSION_MAJOR is not set; check project.mk file)
-endif
-ifndef VERSION_MINOR
-$(error VERSION_MINOR is not set; check project.mk file)
-endif
 
-# Generate version and tag information from inputs
-COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-9]{40}$$"`..HEAD --count)
-BUILD_DATE=$(shell date -u +%Y-%m-%d)
-CURRENT_COMMIT=$(shell git rev-parse --short=8 HEAD)
-CATALOG_VERSION?=$(CHANNEL)-$(BUILD_DATE)-$(CURRENT_COMMIT)
+# Generate version and tag information
+CATALOG_HASH=$(shell find catalog-manifests/ -type f -exec md5sum {} \; | sort | md5sum | cut -d ' ' -f 1)
```

Comment:
`md5sum` isn't portable; on MacOS the binary is `md5` and the output is in a different order by default. Example:

```
~ $ md5 .gitconfig 
MD5 (.gitconfig) = 2034fe26deebdc956ebf5db1d7f684d9
~ $ md5 -q .gitconfig 
2034fe26deebdc956ebf5db1d7f684d9
~ $ md5 -r .gitconfig 
2034fe26deebdc956ebf5db1d7f684d9 .gitconfig
```

`openssl md5` is more portable:

```
~ $ openssl md5 .gitconfig 
MD5(.gitconfig)= 2034fe26deebdc956ebf5db1d7f684d9
```